### PR TITLE
Retrieve messages from channels

### PR DESCRIPTION
**Pull request by iAmao**
### What does this PR do?
This PR adds functionality to retrieve all messages from a channel

### Description of task to be completed
- First makes a get request to the channel.history slack api method, it then parses it as a text and writes it into the messages.json file. Afterwards, it returns the json format of the fetched request.
- By default, the channel it will fetch messages from is the #api_test channel, but if a channel was specified at the command line it fill fetch messages for that channel instead.

### How can this be manually tested?
just run for default `python slack_scrapper.py`
or run `python slack_scrapper.py <channel_id>`

### CODE REVIEW

File: slack_scrapper.py [hunk 1]
```diff
@@ -1,17 +1,29 @@
...
 load_dotenv(dotenv_path)
-
+ 
 # Get the slack token from the .env file
 token = os.environ.get("SLACK_TOKEN")
 
 sc = SlackClient(token)
 
+#Get all messages in a channel
+def allMessages(channel = 'C5GRK45A9'):
```
Comment (BolajiOlajide) [inline slack_scrapper.py:18]: Hey man, can you try this for the #general channel instead, @andela-rbabalola tried this earlier said he could only get about 127 messages at a time. Can you try it for the #general channel and see if you can get more than 127 messages.

Comment (BolajiOlajide) [reply slack_scrapper.py:18]: cc @andela-iamao

File: slack_scrapper.py [hunk 2]
```diff
@@ -49,6 +61,8 @@
         textfile.write("ID: {} \n".format(user["id"]))
         textfile.write("------------------------------------------------ \n")
 
+allMessages(sys.args[1])
```
Comment (BolajiOlajide) [inline slack_scrapper.py:64]: At this point, what is going to be the value of `sys.args[1]` seeing as you've already hardcoded the channel to be scrapped for messages.

**Comment by BolajiOlajide** [comment]
@andela-rbabalola are we merging this?

File: slack_scrapper.py (hunk 2 above)
Comment (rotimi-babalola) [reply slack_scrapper.py:64]: Merging now...

**Pull request merged**